plugin/src/main/kotlin/org/neotech/plugin/rootcoverage/utilities/ProjectExtensions.kt

```diff
@@ -0,0 +1,9 @@
+package org.neotech.plugin.rootcoverage.utilities
+
+import org.gradle.api.Project
+import java.io.File
+
+/**
+ * Returns the output report path composed from the given [fileName] as a [File].
+ */
+fun Project.outputReport(fileName: String): File = file("${project.buildDir}/reports/$fileName")
```

Comment:
Can you rename this function to `Project.getReportOutputFile(fileName: String)`? This is in my opinion a bit more self explanatory, if you look at just the method name:

- `outputReport` "Does this create a report? Does this output a report?"
- `getReportOutputFile` "Ah, so this returns me a file for outputting a report."

Reply:
:heavy_check_mark: Done. Although, I personally would avoid using the return type in the function name. It will make future refactoring harder.